app/test_main.py

```diff
@@ -1,3 +1,59 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "animal_years, human_years",
+    [
+        ([0, 0], [0, 0]),
+        ([14, 14], [0, 0])
+    ]
+)
+def test_below_one_year_returns_zero(
+        animal_years: list,
+        human_years: list
+) -> None:
+    assert get_human_age(*animal_years) == human_years
+
+
+@pytest.mark.parametrize(
+    "animal_years, human_years",
+    [
+        ([15, 15], [1, 1]),
+        ([23, 23], [1, 1])
+    ]
+)
+def test_one_year_range_returns_one(
+        animal_years: list,
+        human_years: list
+) -> None:
+    assert get_human_age(*animal_years) == human_years
+
+
+@pytest.mark.parametrize(
+    "animal_years, human_years",
+    [
+        ([24, 24], [2, 2]),
+        ([27, 28], [2, 2])
+    ]
+)
+def test_two_years_range_returns_two(
+        animal_years: list,
+        human_years: list
+) -> None:
```

Comment:
you test the same function but with different params. you can implement 1 function that will test get_human_age but for each set of test data you can add ids parameter with a test data description